src/SDK/Trace/Span.php

```diff
@@ -361,9 +360,9 @@ public function toSpanData(): SpanDataInterface
             $this->name,
             $this->links,
             $this->events,
-            $this->getImmutableAttributes(),
-            (null === $this->attributes) ? 0 : $this->attributes->getTotalAddedValues(),
-            $this->totalRecordedEvents,
+            $this->attributes->build(),
+            max(0, $this->droppedEventsCount),
```

Comment:
I've spent a while to understand why the count could be negative number. Very interesting approach to store the limit value.

Reply:
Should I revert this change or is adding a comment to the `droppedLinksCount` and `droppedEventsCount` properties sufficient?